# Control Components
uint16 NONE           = 0     # 0x0000
uint16 H_INFINITY_PID = 256   # 0x0100
uint16 CONTROL_TBD    = 512   # 0x0200

# Trajectory Interpolation Components
uint16 NONE           = 0     # 0x0000
uint16 TRAJECTORY_TBD = 1     # 0x0001

uint16 joint_component
uint16 task_component
